apps/labs/posts/conda-grayskull-packaging.md

```diff
@@ -0,0 +1,123 @@
+---
+title: 'Conda and Grayskull, the Masters of Software Packaging'
+published: January 19, 2022
+author: mahe-iram-khan
+description: 'Grayskull is an automatic conda recipe generator, with a focus on conda-forge.'
+category: [Packaging]
+featuredImage:
+  src: ../public/posts/conda-grayskull-packaging/package_name_feature.png
+  alt: 'A flow diagram of a 'package name' being fed into a gray colored skull which represents the tool Grayskull and being converted into a curled up piece of paper which represents a recipe.'
```

Comment:
```suggestion
  alt: 'A flow diagram of a ''package name'' being fed into a gray colored skull which represents the tool Grayskull and being converted into a curled up piece of paper which represents a recipe.'
```

Reply:
in YAML: within single quotes the only supported escape sequence is a doubled single quote (`''`) denoting the single quote itself as in `'don''t'`